build/yaml/steps/windows/nugetrestore.yaml

```diff
@@ -1,22 +1,22 @@
-# Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License.
-
-# Step Template: Restore NuGet Packages
-
-parameters:
-  Solution: ''
-
-steps:
-# NuGet Tool Installer
-# Acquires a specific version of NuGet from the internet or the tools cache and adds it to the PATH. Use this task to change the version of NuGet used in the NuGet tasks.
-- task: NuGetToolInstaller@0
-  inputs:
-    versionSpec: '4.9.4'
-
-# nuget restore
-- task: NuGetCommand@2
-  displayName: Restore ${{ parameters.Solution }}
-  inputs:
-    restoreSolution: ${{ parameters.Solution }}
-    feedsToUse: config
-    nugetConfigPath: NuGet.config
+# Copyright (c) Microsoft Corporation. All rights reserved.

```

Comment:
Why does GitHub say this entire file changed when it seems you really only change the NuGet tool version?

Reply:
Line endings?